tool/src/webapp/js/announcements.js

```diff
@@ -0,0 +1,38 @@
+var number;
+$(function(){
+	number = $('.announcements-div').find($('#numberOfAnnouncements')).val();
+	//set the 'numberOfAnnouncements' in the hidden field for edit screen
+	$('#numberOfAnnouncements').val(number);
+	showAnnouncements();
+});
+
+function showAnnouncements(){
+	var url = $('.announcements-site-url').text().replace(/'/g,"");
+	var announcementsUrl = url + ".json?n=" + number;
+
+	//Get announcements
+	$.ajaxSetup({ cache: false,async: true });	//Disable caching for ajax requests but make asynchronous
+	$.getJSON(announcementsUrl, function(data, status, xhr) {
+		var tool_href = $('.announcements-view-url').text().replace(/'/g,"");
+		//get the announcement tool url
+		var link_to_tool = tool_href.split("?", 1);
+		var text_for_announcements = '<div class="announcementsHeaderDiv"><h3 class="announcementSummaryHeader" style="border-bottom:1px solid #ccc;padding-bottom: 5px;"><a href="'+link_to_tool+'" target="_top" title ="Announcements">Announcements</a></h3></div>';
+		if($(data["announcement_collection"]).size() === 0) {
+			//ie no announcements
+			text_for_announcements = '<p>'+msg("simplepage.announcements-no-message")+'</p>';
+		}
+		else {
+			$(data["announcement_collection"]).each(function(){
+				date = new Date(this["createdOn"]);//*1000);
+				var hour = date.getHours() < 10 ? '0' + date.getHours() : date.getHours();
+				date_time = hour +":"+date.getMinutes()+ " " + date.getDate() + "/"+ (date.getMonth()+1) + "/"+ date.getFullYear();// create a new javascript Date object based on the timestamp
```

Comment:
Is this all ok with relation to timezones? Sakai has the concept of a timezone, does the date returned by entitybroker include the user's timezone?